channel/backend.go

```diff
@@ -62,26 +62,14 @@ func (Backend) Verify(addr wallet.Address, state *channel.State, sig wallet.Sig)
 	return wallet.VerifySignature(buf.Bytes(), sig, addr)
 }
 
-// NewAsset returns an empty noAsset.
+// NewAsset returns a fabric asset.
+// Does not contain any fields since there is only one asset per chain.
```

Comment:
What do you mean with this comment? If it refers to that the asset itself does not have any fields, then I would add this comment to the definition of the asset.

Reply:
You are right. Actually this comment is already on the definition of `asset`.